torchtitan/experiments/deepseek_v3/README.md

```diff
@@ -0,0 +1,39 @@
+# Running DeepSeek in Titan  (experimenatal)
+
+This folders contains a DeepSeek model and kernels and scripts needed to run it.
+
+## Inference
+
+### Prerequisites:
+
+You will need to download a DeepSeek model's weights if you want to run a
+pre-trained checkpoint.  We provided a script to download the weights from
+HuggingFace Model Hub:
+```bash
+python download.py [vX]
+```
+where `vX` can be v2 or v3, both are supported. You may be required to create a
+HuggingFace account and log in first.
+
+### Running inference:
+
+The inference script is in `generate.py`. You can run it with the following
+command:
+```bash
+torchrun --standalone --nproc-per-node 4 generate.py
+```
+This will run inference on the `DeepSeek-V2-Lite-Chat` model using 4 GPUs by
+default.
+
+Alternatively, you can run inference by using `bash inference.sh`.
```

Comment:
Alternatively, you can run inference by using `bash inference.sh `, optionally followed by your prompt.